ocaml/idl/datamodel.ml

```diff
@@ -562,6 +562,19 @@ let _ =
   error Api_errors.user_is_not_local_superuser [ "msg" ]
     ~doc:"Only the local superuser can execute this operation" ();
 
+  (* SR-IOV errors *)
+  error Api_errors.network_sriov_already_enabled ["PIF"]
+    ~doc:"You tried to enable network SR-IOV on PIF which is already enabled" ();
+  error Api_errors.network_sriov_find_pf_from_vf_failed ["PCI"]
+    ~doc:"Failed to find physical function from virtual function" ();
+  error Api_errors.network_sriov_pif_has_vf_inuse ["PIF"; "VFs"]
+    ~doc:"Can not destroy network sriov object which has virtual function inuse" ();
+  error Api_errors.network_sriov_enable_failed ["PIF"; "msg"]
+    ~doc:"Can not enable network SR-IOV on PIF" ();
```

Comment:
`Can not -> Cannot`, but I would actually write `"Failed to enable SR-IOV network."`